projects/aca-content/preview/src/lib/components/preview.component.html

```diff
@@ -28,9 +28,7 @@
     </adf-viewer-open-with>
 
     <adf-viewer-toolbar-actions *ngIf="!simplestMode">
-      <ng-container *ngFor="let action of viewerToolbarActions; trackBy: trackByActionId">
-        <aca-toolbar-action [actionRef]="action"></aca-toolbar-action>
-      </ng-container>
+      <aca-toolbar [items]="viewerToolbarActions"></aca-toolbar>
```

Comment:
Small thing but I'm wondering if that naming is not a bit confusing? Because looking at structure we have aca-toolbar in adf-viewer-toolbar-actions -> 'toolbar' in 'toolbar actions'. It looks a bit strange that we have toolbar in toolbar actions and not toolbar actions in toolbar...

Reply:
ok fine, let's leave it